### PR TITLE
fix: crash when answering a call (WPB-6183)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6183" title="WPB-6183" target="_blank">BugWPB-6183</a>  [Android] crashes during 2024-01-17 playtest answering calls
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Crash when answering a call caused by a null name.
details here:
https://github.com/wireapp/kalium/pull/2481

### Solutions

This PR is handling null name case by displaying a default value.

### Dependencies (Optional)

Should be merged after
https://github.com/wireapp/wire-android/pull/2679
https://github.com/wireapp/kalium/pull/2481

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [3041](https://10.10.124.134/job/AR-build-pipeline/3041/) **failed**.

**Comment by AndroidBob** [comment]
Build [3049](https://10.10.124.134/job/AR-build-pipeline/3049/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.7.0(8876112)(8876112)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.7.0(8876112)(8876112)-staging-compat.apk)

**Review by saleniuk — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/calling/ongoing/participantsview/ParticipantTile.kt [hunk 1]
```diff
@@ -92,7 +92,8 @@ fun ParticipantTile(
     isSelfUserMuted: Boolean,
     isSelfUserCameraOn: Boolean,
     onSelfUserVideoPreviewCreated: (view: View) -> Unit,
-    onClearSelfUserVideoPreview: () -> Unit
+    onClearSelfUserVideoPreview: () -> Unit,
+    defaultUserName: String = "DEFAULT"
```
Comment (mchenani) [inline app/src/main/kotlin/com/wire/android/ui/calling/ongoing/participantsview/ParticipantTile.kt]: Question: how about translation? Maybe it's better to extract it to strings.

**Review by mchenani — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [3109](https://10.10.124.134/job/AR-build-pipeline/3109/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.7.0(8887367)(8887367)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.7.0(8887367)(8887367)-staging-compat.apk)

**Pull request merged**